packages/foundation/src/Foundation.test.ts

```diff
@@ -403,4 +403,37 @@ describe('Given the foundation plugin', () => {
       expect(request.value).toBe(value)
     })
   })
+
+  describe('getExternalUrl function', () => {
+    test('should return the correct url for a 721 mint', async () => {
+      const mint: MintActionParams = {
+        chainId: Chains.BASE,
+        contractAddress: '0xead6dca70b0465725a57eb81f7d3ab8b5e0b81b4',
+      }
+      const result = await getExternalUrl(mint)
+      expect(result).toBe(
+        'https://foundation.app/mint/base/0xead6dca70b0465725a57eb81f7d3ab8b5e0b81b4',
+      )
+    })
+
+    test('should return the base url for 1155 mint', async () => {
+      const mint: MintActionParams = {
+        chainId: Chains.BASE,
+        contractAddress: '0x1d2550d198197df1a10af515cf2ea0d790889b93',
+        tokenId: 213,
+      }
+      const result = await getExternalUrl(mint)
+      expect(result).toBe('https://foundation.app/')
```

Comment:
This test result doesn't look correct? Shouldn't it also include the action params in here?

Reply:
Nvm I see that this is intentional. Is it possible to extract the url from Foundation? Or will it be a super deep dive into their API or something?

Reply:
There erc-1155 implementation is custom and has no available frontend. The last one was for their babes collection.

https://babes.foundation.app/

Reply:
It might be available in the contract or token uri

Reply:
No luck.